app/src/test/java/sleepyweasel/purplefluffernutter/ContentProviderTest.java

```diff
@@ -0,0 +1,105 @@
+package sleepyweasel.purplefluffernutter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.Assert.*;
+
+public class ContentProviderTest {
+
+    private MovieContentProvider contentProvider = MovieContentProvider.getInstance();
+
+    private static final String MOVIE_TITLE_1 = "title 1";
+    private static final String MOVIE_TITLE_2 = "title 2";
+
+    private MovieEntry firstEntry = new MovieEntry(MOVIE_TITLE_1);
+    private MovieEntry secondEntry = new MovieEntry(MOVIE_TITLE_2);
+
+    @Before
+    public void setUp() throws Exception {
+        contentProvider.clear();
+    }
+
+    @Test
+    public void shouldCreateEmptyContentProvider() throws Exception {
+        assertThat(contentProvider.isEmpty()).isTrue();
+    }
+
+    @Test
+    public void shouldAddMovieEntryToContentProvider() throws Exception {
+        contentProvider.addMovie(firstEntry);
+
+        assertThat(contentProvider.isEmpty()).isFalse();
+    }
+
+    @Test
+    public void shouldClearContentProvider() throws Exception {
+        contentProvider.addMovie(secondEntry);
+
+        contentProvider.clear();
+
+        assertThat(contentProvider.isEmpty()).isTrue();
+    }
+
+    @Test
+    public void shouldReturnSizeZeroIfContentProviderEmpty() throws Exception {
+        assertThat(contentProvider.size()).isEqualTo(0);
```

Comment:
to jak mam to zmienić, poza tym, że isZero()? Przed każdym testem tworzony jest na początku pusty provider